### ISSUE TITLE
Cannot search contacts by phone number

### ISSUE THREAD

**Issue by mikeashley** [owncloud/contacts#797]
Searching contacts by name works. Searching contacts by phone number does not appear to work. Practical use case: I get a text or phone call from an unrecognized number on my (not-smart) phone. I want to see if it's from somebody I know.

**Comment by gstuartj** [comment]
You're right, according to provider.php you can search by name, organization, email, or nickname. It would be very useful to add the ability to search by phone number.
